Makefile

```diff
@@ -0,0 +1,24 @@
+
+default: bin
+
+.PHONY: all
+all:  gomod_tidy gofmt bin test
+
+.PHONY: gomod_tidy
+gomod_tidy:
+	 go mod tidy
+
+.PHONY: gofmt
+gofmt:
+	go fmt -x ./...
+
+.PHONY: bin
+bin:
+	 go build *.go
+	 hack/build.sh
+
+.PHONY: test
+test:
+	 go test ./...
```

Comment:
Can we add `-v` flag for verbose output?

Reply:
Added.